### DISCUSSION TITLE
The original solution does not seem to converge

### DISCUSSION THREAD

**Discussion by xbbgq** [mdolab/dafoam#443] [General]
I use the same BCs as rotor37, because it is two-dimensional, I set the hub and shroud to be symmetryPlane.The original solution does not seem to converge
![1](https://github.com/mdolab/dafoam/assets/87130278/5f0a69bc-ad52-410e-abf5-3305585baaee)
/*--------------------------------*- C++ -*----------------------------------*\
| =========                 |                                                 |
| \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox           |
|  \\    /   O peration     | Version:  v1812                                 |
|   \\  /    A nd           | Web:      www.OpenFOAM.com                      |
|    \\/     M anipulation  |                                                 |
\*---------------------------------------------------------------------------*/
FoamFile
{
    version     2.0;
    format      ascii;
    class       volScalarField;
    location    "0";
    object      p;
}
// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //

dimensions      [1 -1 -2 0 0 0 0];

internalField   uniform 101325;

boundaryField
{
    blade
    {
        type            zeroGradient;
    }
    inlet
    {
        type            fixedValue;
        value           uniform 101325;
    }
    outlet
    {
       type            fixedValue;
        value           uniform 125000;
    }
    shroud
    {
        type          symmetryPlane;
    }
    hub
    {
        type          symmetryPlane;
    }
    per1
    {
        type            cyclic;
    }
    per2
    {
        type            cyclic;
    }
}

**Comment by friedenhe** [comment]
If the flow did not converge, you need to play with the BCs. If the adjoint did not converge, check the DAFoam website FAQ.

**Reply by xbbgq** [reply]
I compared the result cloud map of FLUENT, the flow did not converge. Do I need to change system configuration files?

**Reply by friedenhe** [reply]
Yes, you may need to tweak the files in the 0, constant, and system folders. Ideally, you should converge it with the native OpenFOAM solver and make sure the flow configurations work, before running DAFoam optimization. If it is a flow convergence issue, I suggest you post this to CFDonline.com
